server/app/lib/utils/fusor/mac_addresses.rb

```diff
@@ -0,0 +1,16 @@
+module Utils
+  module Fusor
+    class MacAddresses
+
+      def self.generate_mac_address
+        options = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'A', 'B', 'C', 'D', 'E', 'F']
+        l = [[options.sample, '2'].join('')]
+        for i in 0..4
+          l << (options.sample 2).join('')
+        end
+        l.join(':')
+      end
+
+    end
+  end
+end
```

Comment:
We should add a test for MacAddresses class. Even if it's a trello card for now.